[2.8] [PR 95] Make the doc URL to use \"next\" when the branch is master (#98)
owncloud/docs-client-desktop: bin/instructions_makepdf.md

```
= ownCloud Administration Manual
:toc:
:toclevels: 2
:homepage: https://github.com/owncloud/docs
:icon-set: octicon
:icons: font
:listing-caption: Listing
:source-highlighter: rouge
:version-label: Version:
:module_base_path: modules/admin_manual/pages/
The ownCloud Team <docs@owncloud.com>
{revnumber}, {revdate}
```
You can adjust the settings as needed. `:module_base_path:` is the path to the base document files of the buildable manuals and must be set accordingly. The script takes the chosen template file, adds based on the table of contents of the manual (`nav.adoc`) include statemens like below
```
include::{module_base_path}index.adoc[leveloffset=+1]

include::{module_base_path}installing.adoc[leveloffset=+1]
...
```
and pipes this as source to the processor. You can define the level how deep you want to render the table of contents with `:toclevels:`. The pdf file created locally is saved in the `pdf_web` directory with a naming defined by the script.

When using the debug mode `-d` of `makepdf` as described above, you see the files specified in the base document to use when generating a pdf.

## Create a New Documentation Repo

> In most cases it is a good idea to copy the contents of an existing repo and adopt them to the needs. In particular you need to take care on following folders and files.

Reviewer: ```suggestion
In most cases it is a good idea to copy the contents of an existing repo and adopt them to your needs. In particular, you need to take care of the following folders and files.
```

> - Provide book template(s) .adoc file in the `book_templates` directory, adapted them to the requirements.

Reviewer: ```suggestion
- Provide book template(s) .adoc file in the `book_templates` directory and adjust it as required.
```

- Copy the `pdf_web`, `fonts` and `resources` directory.

> - Make a copy of an existing `manual_config` file which must reside on the same level of the makepdf file and adopt its contents according the setup of the new repo. See the description inside the file. It will define the parameters the makepdf script will run for this repo.

Reviewer: ```suggestion
- Make a copy of an existing `manual_config` file which must reside on the same level of the makepdf file and adjust its contents according to the setup of the new repo. See the description inside the file. It will define the parameters the makepdf script will run for this repo.
```
